test/docker-test/omsfiles/customlog.conf

```diff
@@ -4,7 +4,7 @@
   type sudo_tail
   path /var/log/custom.log
   pos_file /var/opt/microsoft/omsagent/state/CUSTOM_LOG_BLOB.Custom_Log_CL_<workspace-id>.pos
-  read_from_head false
```

Comment:
FYI @jamvar this caused a regression where oracle7 no longer had custom logs

Reply:
so i have reverted

Reply:
But, It has to be set to false. This change is an update and will be available in all regions next week. Going forward this will be the setting. I did see that oracle 7 custom log is not working with this set to false, but let's keep it at false and try to make it work.